src/site/content/en/blog/aem-with-workbox/index.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: A modern web experience on Adobe Experience Manager with WorkBox
+description: >
+  Progressive Web Apps leverage what the modern web can do. Adobe used Workbox to bring those capabilities to Adobe Experience Manager.
+authors:
+  - joemedley
+date: 2022-03-04
+hero: image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/RKF1rl4ItVpoMj7TRZHG.png
+alt: The Adobe Experience Manager logo.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - case-study
+---
+
+If you are a technical lead or digital marketing analyst interested in providing a modern Web experience to your Adobe Experience Manager (AEM) web application and have been looking for your options in doing so, then you have come to the right article. This will cover what Progressive Web Apps (PWA) are and what you need to create a PWA in AEM leveraging the WorkBox library through configuration, without coding.
+
+## Why PWA?
+
+Progressive Web Apps leverage what the modern web can do. They are installable on your device, load quickly, with subsequent visits loading instantly. They respond to input quickly. They work well on an unreliable connection or when offline. PWAs use modern APIs to provide an engaging app-like experience with an optional full screen UI, background updating, and offline access to data.
+
+{% Img src="image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/OktJivHyntcGzX1VSdym.png", alt="From web app to Progressive Web App.", width="800", height="477" %}
```

Comment:
```suggestion
<figure>
  {% Img src="image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/OktJivHyntcGzX1VSdym.png", alt="From web app to Progressive Web App.", width="800", height="477" %}
</figure>
```

Reply:
Just a suggestion, but I tend to wrap figures in `<figure>` elements. If there's no good reason for this, feel free to reject this and similar suggestions.